api/src/Microsoft.Azure.IIoT.OpcUa.Api.Publisher/src/Models/PublishedNodesResponseApiModel.cs

```diff
@@ -4,23 +4,11 @@
 // ------------------------------------------------------------
 
 namespace Microsoft.Azure.IIoT.OpcUa.Api.Publisher.Models {
-    using System.Collections.Generic;
-    using System.ComponentModel.DataAnnotations;
-    using System.Runtime.Serialization;
-    
 
     /// <summary>
     /// PublishNodes direct method response
     /// </summary>
-    [DataContract]
     public class PublishedNodesResponseApiModel {
 
-        /// <summary>
-        /// Status messages
-        /// </summary>
-        [DataMember(Name = "statusMessage", Order = 0)]
-        [Required]
-        public List<string> StatusMessage { get; set; }
```

Comment:
Don't we at least want to keep one string to communicate exceptions ? Can you please also change the E2E tests in this PR as well so that we see hot this ends up on that end as well ? Would also be useful if we add some calls in E2E tests that we know should throw exceptions, such as removing nonexistent endpoints, and validate the responses in that cases.

Reply:
The DM call exceptions are handled exclusively by throwing `MethodCallStatusException`. This one has a field for description besides the (Bad) status. 
Right, extending E2E with bad things is a good idea.